HyperPlay-Gaming/hyperplay-desktop-client
# [Tech] Use DNS text queries to resolve PCI IDs (#3528)

In short: does not rely on pcids file anymore, gets the info, cache it and use it for 30 days until the cache expires.

Those Ids are used to detect the user system hardware to show on the log screen and logs.

Original PR:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/3528

> * Use DNS text queries to resolve PCI IDs
> 
> We only want to clear them if we're *not* upgrading Heroic though (don't
> want to cause a request spike when releasing an update)


### To test

Just check if the system information shows fine on the logs screen.

---

Use the following Checklist if you have changed something on the Backend or Frontend:
